agent/bench-scripts/postprocess/fio-postprocess

```diff
@@ -257,17 +257,25 @@ foreach $client_hostname_dir (@client_hostname_dirs) {
 					my $timestamp_ms = $1;
 					my $value = $2;
 					my $rwtype = $3;
-					my %sample = (get_label('date_label') => int $timestamp_ms, get_label('value_label') => int $value, get_label('rw_label') => int $rwtype);
+					my %sample;
 					my $index = value_exists(get_label('date_label'), $timestamp_ms, \@samples);
-					# duplicate timestamp *AND* duplicate rwtype
-					if ($index >= 0 && $samples[$index]{get_label('rw_label')} == $rwtype) {
-						printf "warning: timestamp already exists with this RW type, skipping this sample (file = %s timestamp = %d, value = %d, rwtype = %d)\n",
```

Comment:
One thing occurred to me about this warning: I added this because we observed (sadly quite often) that fio would report an operation twice with the same timestamp but different values.  I am a little concerned that we are losing the ability to identify this problem.  Also, I am wondering, if there are more than 1 sample for the same time-stamp [and same rw-type] are we handling this properly?  My understanding is that below we are just adding to the value (line 266).  If we encounter >1 sample with same timestamp and same rwtype, we could be adding those two values.  That is going to be the wrong interpretation.  We either need to drop duplicate samples [of the same rw-type and timestamp] or average all of those samples' values into 1 value.  We should only combine values when the rw-types are different.  So I think you will still need to track $rwtype while samples are being processed, but the sample itself does not have to have a $rwtype.

Reply:
Ugh.  What do you think is more appropriate, ignoring the values for repeated timestamp+rwtype combinations (the previous behavior) or averaging the multiple values for the same combo?

Reply:
I suppose more appropriate is averaging.

Reply:
Perhaps we can add a flag to that number to indicate it is an average of two values having the same timestamp?

Reply:
I don't think adding a flag will work without a fairly complete modification of the entire postprocessing stack to surface new properties like that.  Further, as it turns out this is a fairly common issue so I don't think it is the kind of thing that will happen once and awhile and you want to know.  It happens all the time.  My new approach does provide more verbose logging that can be used to investigate the values if the user desires.